dc/s2n-quic-dc/src/stream/client/tokio.rs

```diff
@@ -65,8 +87,69 @@ where
     H: core::future::Future<Output = io::Result<secret::map::Peer>>,
     Sub: event::Subscriber + Clone,
 {
-    // Race TCP handshake with the TLS handshake
```

Comment:
Can you keep this comment? It is useful for people who don't know the codebase.

Also, for my own knowledge, what specifically is being raced here? TCP + TLS with QUIC + TLS? Because just racing TCP against TLS doesn't really make any sense.